cmd/kops/create_cluster.go

```diff
@@ -314,9 +314,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	// TODO: Can we deprecate this flag - it is awkward?
 	cmd.Flags().BoolVar(&associatePublicIP, "associate-public-ip", false, "Specify --associate-public-ip=[true|false] to enable/disable association of public IP for master ASG and nodes. Default is 'true'.")
 
-	if featureflag.AWSIPv6.Enabled() {
-		cmd.Flags().BoolVar(&options.IPv6, "ipv6", false, "Allocate IPv6 CIDRs to subnets for clusters with public topology (AWS only)")
-	}
+	cmd.Flags().BoolVar(&options.IPv6, "ipv6", false, "Allocate IPv6 CIDRs to subnets for clusters with public topology (AWS only)")
```

Comment:
This description should be improved. It's more around the pods being IPv6-only. And the "public topology" part is out of date.

Perhaps something like:

```suggestion
	cmd.Flags().BoolVar(&options.IPv6, "ipv6", false, "Make the pod network family be IPv6 (AWS only)")
```

But perhaps we should have a `--network-family` flag with values `ipv4` (default) and `ipv6`?

I've seen references from AWS to IPv6-only subnets, so we should think about how we'll handle those.

Reply:
I agree on the message.
About the flag, we can discuss it during the next office hours. 1.23 is still alpha so, not really an issue.